languages/python/exercises/concept/none/.docs/introduction.md

```diff
@@ -0,0 +1,26 @@
+When you are beginning to code, you always assign a variable to a value. Say,
+
+```python
+a = 3
+b = True
+```
+
+This declares the variable as well as defines them. But when you want to just declare the variable and not define them at start you use a special built in called as `None`.
+
+If you have heard about `NULL` type in other languages then this must be familiar. `None` helps you to declare variables. These variables can be used later to be defined to a value.
+
+```python
+a = None
+print(a)
+#=> None
+type(a)
+#=> <class 'NoneType'>
+```
+
+In Python, `None` is a Singleton built-in which is used to declare a variable as well as can be used as `False`. Meaning if you want to check for a variable which is declared with `None`, It will be considered as `False`
+
```

Comment:
I think it's good here to emphasize the "placeholder"/null nature of `None` -- that it represents an _**absence of value**_,  and also to mention that Python uses `None` when a return from a function is not specified.    But referring to variable declaration (_None can be a placeholder when you don't have a default value, but it really doesn't have a function beyond that_) confuses things here.  -- see [python docs](), [https://docs.python.org/3/library/constants.html#None], [David Beazly's Python Course](https://dabeaz-course.github.io/practical-python/Notes/02_Working_with_data/01_Datatypes.html) (_which mentions the conditional evaluation_), and  [educative.io None keyword](https://www.educative.io/edpresso/what-is-the-none-keyword-in-python) for some wording examples that are a little more direct.

 I also think it's dangerous to tell the student that `None` can be _**used as**_ `False` as a main case.  `None` _**evaluates**_ to `False` when used in a conditional check (_which is something we should cover in this exercise_), but `None` is not of type `bool`, and won't behave as such.  For cases where a `False` type is needed, `bool` type `False` should be used.  "Falsey" and "Truthy" are a different concept exercise.

Reply:
I also think we should save the discussion about `None` being a Singleton object for the `after.md` (_maybe link to the educative.io page, or some other good explanation??_).  It is very valuable information, but not really necessary for understanding basic usage or for completing this exercise.

Reply:
Got it. I will mode the discussion of None being a singleton object to after.md 

It is something very necessary to know but also a little tough to understand. I will refer some docs to bring it up more easier.

Reply:
Agree.  We might want to consider a follow-on concept exercise around `None` being a singleton, or its possible uses as a `sentinel value` -- but it feels like that might be under an OOP category, or some other more "advanced" scenario.  

But speaking to those uses in the `after.md` is a great start -- you can give examples, point the student to some articles or problems as "next steps", or link to projects or code where `None` is used in that fashion.  There are a lot of possibilities for what a student could explore after this basic run-through....we just don't want to chase them away with too much info at the very start.  😉